infrastructure/stacks/roles/locals.tf

```diff
@@ -0,0 +1,15 @@
+locals {
+
+
+  fuzzy-search-dynamodb_access_iam_name = "${var.service_tag}-${var.profile}-${var.project_tag}-dynamodb-access"
+
+  dynamoDb_full_access_policy_arn      = "arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess"
```

Comment:
Fuzzy API only needs read-only access. Is there an option for this?  Also, there should be a way of only allowing the API to reach our DynamoDB

Reply:
So I added the read only dynamoDB out the box policy, however I then couldnt lock it down to a specific table. So what I have done is I have copied the read only policy and created a new one, but locked that new one down to our specific table.